### DISCUSSION TITLE
Can apply the CIS Compliance package AFTER buying and deploying the Ref Arch?

### DISCUSSION THREAD

**Discussion by marijakstrazdas** [gruntwork-io/knowledge-base#316] [Help] [answered]
Can apply the CIS Compliance package AFTER buying and deploying the Ref Arch?

**Comment by yorinasub17** [comment, marked as answer]
We do not currently have a migration guide that helps transition from a non-CIS reference architecture to a CIS reference architecture. Although this is doable, the steps are fairly involved (due to the remapping of resource addresses) and we highly recommend starting with the CIS reference architecture if you are contemplating CIS compliance.

With that said, if you do decide that you want to take this path of migration, we are happy to help out in support as needed. But please be aware that the support we can offer will be limited by the different support tiers that you are on.
